runtime/oti/j9consts.h

```diff
@@ -217,7 +217,9 @@ extern "C" {
 #define J9_METHOD_ENTER_RASTRACE 0x10
 #define J9_METHOD_ENTER_NEW_HOOK_INTERFACE 0x20
 
+/* The method has been queued for JIT compilation */
```

Comment:
These comments should be in the big comment, not here.

Reply:
To me this documentation describes what this particular define represents. The fact that it can get stored in `J9Method.extra` is not really relevant, nor does it change the description of the define. For example if we see a use of this define by itself, such as:

https://github.com/eclipse/openj9/blob/7d909ad90e3ce2fd25d44e797e946e24308bb6c5/runtime/compiler/control/CompilationThread.cpp#L5165-L5169

One doesn't have to dig through layers of code to know that the meaning of the define is documented in the `J9Method.extra` field. One simply has to go to the definition of the define itself to understand what it means.

Does this make sense? Do you agree?

Reply:
Personally, I think the name is self-descriptive anyway. It doesn't make sense to me that the new doc says to go look somewhere else for the meanings.

Reply:
Also worth noting the "1" in the code above...

Reply:
I've fixed the uses of "1" in the above code in bea8acc. They are not wrong, but simply not using the right defines and chose to use magic constants instead. @dsouzai please review the latest commit.

Reply:
@gacholio I've fixed up the documentation by inlining it into j9nonbuilder.h and adding documentation for the default method conflict case in a [force-push](https://github.com/eclipse/openj9/compare/bea8acc31ea7bca90da764cc0204978003bbd962..bee1eebce5b105bc6ad9d3350b4371bd993313c3). Personally I don't think we should be using `J9_STARTPC_NOT_TRANSLATED` for the default method case though as we cannot distinguish this from the count case. IMO we should be using a different define value for that. The `J9Method*` is aligned, so we can use for example 0x2 to tag the `extra` field for default method conflicts. Thoughts on that?

Reply:
The default conflict case will never be passed to the JIT (the send target will always throw an exception immediately), so there's probably no need for a tag at all - it was just there for consistency. Introducing a new tag would serve no possible purpose.

As this is really a documentation-only change, I suggest not changing anything.

Reply:
I can imagine a scenario where compiled code wants to run the conflict method (to throw the exception), so tagging it as not compiled would be essential in that case (forcing the compiled code to go to the interpreter to run the method).